### ISSUE TITLE
Input is broken for popups in floating views

### ISSUE THREAD

**Issue by emersion** [swaywm/sway#2492]
1. Open gnome-dictionary
2. Make it float, resize it to be small
3. Open the popover menu
4. Cannot click items outside the window geometry

**Comment by sharethewisdom** [comment]
My problem with mupdf might be related. I tried downgrading several versions of mupdf to be sure. Although the window isn't floating, I can only rarely (1 time out of 30 or so) make mupdf react to key presses when the window is focused. Scrolling using the mousepad works. I used the stock config, `/etc/sway/config` and `sway --validate` shows no errors. I it with a new user across several reboots.

```
Linux libreboot 4.18.1-arch1-1-ARCH #1 SMP PREEMPT Wed Aug 15 21:11:55 UTC 2018 x86_64 GNU/Linux`
sway version 1.0-alpha.5-114-g389d159c (Aug 19 2018, branch 'master')
```

I'll try to debug, but have little time.

ps.: I also have been experiencing issues with Firefox: clicks sometimes suddenly cease all interaction on the webpage and the toolbar. The context menu works, though.
